package.json

```diff
@@ -28,18 +28,6 @@
     "type": "git",
     "url": "https://github.com/apiaryio/dredd"
   },
-  "babel": {
-    "presets": [
-      [
-        "env",
-        {
-          "targets": {
-            "node": "4.0.0"
-          }
-        }
-      ]
-    ]
-  },
```

Comment:
Wondering whether @realityking wasn't right about setting the target to 6.0.0 instead of removing it altogether.